api_v2.json

```diff
@@ -1732,11 +1733,38 @@
 						"description": "**POST /v2/payments/{id}/delete ** \n   Delete a prevoiusly created payment and returns a hash containing the details of the deleted payment.\n\n\n**Required Fields**\n\nid:Unique ID of payment\n\n**Optional Fields**\n\non\\_behalf\\_of: contact ID\n\n**Response**\n\nId:Payment unique ID\n\nPayer\\_id:Unique payer ID\n\nAmount:Amount of Payment to 2dp\n\nBeneficiary\\_id:ID of the beneficiary\n\nCurrency:3 character ISO 4217 currency code\n\nConversion\\_id:Conversion unique ID\n\nReference:Reference\n\nPayment\\_type:Can be standard or local\n\nPayment\\_date:ISO 8601 Date when the payment should be paid\n\nreason\t:Reason for payment\n\nstatus:Status of the payment\n\ntransferred\\_at:ISO 8601 Date when the Payment was sent\n\nauthorisation\\_steps\\_required:Number of authorisation steps required\n\ncreator\\_contact\\_id:ID of the contact\n\nlast\\_updater\\_contact\\_id:ID of the contact\n\nshort\\_reference:transfer ID of the payment\n\nfailure\\_reason:Reason for the failure\n\ncreated\\_at:Datetime of when the Payment was created\n\nupdated\\_at:Datetime of when the Payment was last updated\n\n\n"
 					},
 					"response": []
+				},
+				{
+					"name": "Payment Confirmation",
+					"request": {
+						"method": "GET",
+						"header": [],
+						"body": {},
+						"url": {
+							"raw": "{{url}}payment/{{payment_id}}/confirmation",
```

Comment:
here should be {{id}} which is payment id

Reply:
@oladan call it payment_id to be more clear

Reply:
got it